### PR TITLE
Changes to et.json

### PR DESCRIPTION
**Pull request by Repsionu**
Best, JR

### CODE REVIEW

File: src/translations/et.json [hunk 1]
```diff
@@ -2,40 +2,41 @@
...
-    "automatic": null,
-    "recommended": null,
-    "override_headline": null
+  "entity": "olem",
+  "integration": "sidumine",
+  "required": "nõutav",
+  "automatic": "vaikimisi",
+  "recommended": "soovituslik",
+  "override_headline": "pealdise alistamine"
   },
```
Comment (MrBartusek) [inline src/translations/et.json:21]: Invalid indent

File: src/translations/et.json [hunk 2]
```diff
@@ -2,40 +2,41 @@
...
     "name": "Ohtlike ilmanähtuste kaart",
-    "description": "Ohtlike ilmanähtuste kaart hoiatab ilmaohtude eest.",
+    "description": "Äärmuslike ilmaolude hoiatus.",
     "not_available": "Meteoalarm pole saadaval"
   },
   "error": {
     "missing_entity": "Vajalik on Meteoalarmi olem!",
-    "missing_integration": null,
-    "invalid_integration": null
+
```
Comment (MrBartusek) [inline src/translations/et.json:10]: No empty line needed

File: src/translations/et.json [hunk 3]
```diff
@@ -2,40 +2,41 @@
...
+    "rain_flood": "Paduvihm"
   },
   "messages": {
-    "yellow": "Kollane hoiatus {0}",
-    "orange": "Oranž hoiatus {0}",
-    "red": "Punane hoiatus {0}"
+    "yellow": "{0}",
+    "orange": "{0}",
+    "red": "{0}"
   }
```
Comment (MrBartusek) [inline src/translations/et.json:41]: As I said, if you think that card is not properly displayed please create an issue indeed of changing translation files

**Review by MrBartusek — CHANGES_REQUESTED** [review]

**Comment by Repsionu** [comment]
Thanks, next time will try not to mess up...

**Pull request closed without merging**